common/app/views/support/HtmlCleaner.scala

```diff
@@ -942,7 +940,9 @@ object AffiliateLinksCleaner {
     element.tagName == "a" && SkimLinksCache.isSkimLink(element.attr("href"))
 
   def insertAffiliateDisclaimer(document: Document, contentType: String): Document = {
-    document.body().append(affiliateLinksDisclaimer(contentType).toString())
+    if (contentType == "gallery") {
+      document.body().prepend(affiliateLinksDisclaimer(contentType).toString())
```

Comment:
Note to future archaeologists: for legal reasons, we are now prepending the disclaimer to the article body, not appending it.